src/components/common/Extensions/UserHub/partials/BalanceTab/partials/StreamsInfoRow/utils.ts

```diff
@@ -95,11 +133,17 @@ export const calculateTotalsFromStreams = async ({
         colony,
       });
 
+      const isItemsCountedToAverage = isDateEarlierThanOneYear(
```

Comment:
This is about the current item, right? If so could you change the name to singular `item`?